django/db/backends/oracle/introspection.py

```diff
@@ -60,6 +63,13 @@ def get_table_list(self, cursor):
 
     def get_table_description(self, cursor, table_name):
         "Returns a description of the table, with the DB-API cursor.description interface."
+        # user_tab_columns gives data default for columns
+        cursor.execute("""
+            SELECT column_name, data_default
+            FROM user_tab_cols
+            WHERE table_name = UPPER(%s)""", [table_name])
+        columns_default = {line[0]: line[1] if line[1] != 'NULL' else None for line in cursor.fetchall()}
```

Comment:
Using tuple unpacking such as `... for a, b, ... in cursor.fetchall()` rather than indexing should help readability.